packages/wallets/src/components/Base/WalletButtonGroup/WalletButtonGroup.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { Children } from 'react';
+import './WalletButtonGroup.scss';
+import classNames from 'classnames';
+
+type TWalletButtonGroupProps = {
+    isVertical?: boolean;
+};
+
+const WalletButtonGroup: React.FC<React.PropsWithChildren<TWalletButtonGroupProps>> = ({ children, isVertical }) => {
+    return (
+        <div
+            className={classNames('wallets-button-group', {
+                'wallets-button-group--vertical': isVertical,
+            })}
+        >
+            {Children.map(children, child => (
+                <div className='wallets-button-group__item'>{child}</div>
```

Comment:
```suggestion
                <div className='wallets-button-group__item'>{child}</div>
```
This will throw a react error on the key. Can you generate something unique? Or use nanoid (worst case)

Reply:
Best to use CSS for this